source/Test.cpp

```diff
@@ -128,6 +215,16 @@ void Test::LoadSequence(const DataNode &node)
 					step.jumpOnFalseTarget = child.Token(2);
 				step.conditions.Load(child);
 				break;
+			case TestStep::Type::CALL:
+				if(child.Size() < 2)
+				{
+					status = Status::BROKEN;
+					child.PrintTrace("Error: Invalid use of \"call\" without name of called (sub)test:");
+					return;
+				}
+				else
+					step.nameOrLabel = child.Token(1);
+				break;
```

Comment:
The current approach gives a "test not found" at test-execution when a test was missing, while registering with GameData would give a "test not loaded" (but found since we registered it) during test-execution. There could be a minor performance benefit because the translation from string-to-test would happen at test-loadtime instead of test-runtime, but I don't think it makes a lot of difference.

Let's keep this as-is for now. (We can always improve this in a later PR, and then we might want to include the INJECT referencing below as well.)